src/app-layer-smtp.c

```diff
@@ -284,6 +287,67 @@ static void SMTPConfigure(void) {
             smtp_config.mime_config.extract_urls = val;
         }
 
+        extract_urls_schemes = ConfNodeLookupChild(config, "extract-urls-schemes");
```

Comment:
I find it hard to parse this... what is happening here on a high level (please add comment)